Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN npm install --production
 
 COPY . .
 
+RUN chmod +x ./wait-for
```

Comment:
Is it possible to avoid having to make it an executable inside the Dockerfile?

E.g.: Could it just be an executable?

Reply:
I've replaced the wait-for script with the following script:

```
command: > 
  sh -c "
    while ! nc -z db 5432; do
      echo 'Postgress is unavailable.'
      sleep 1
    done
    npm run start"
```

What do you think?